src/workflow/internal/ApiTokenClientInterceptor.ts

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright 2024 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import * as grpc from "@grpc/grpc-js";
+import { Settings } from "../../utils/Settings.util";
+
+export function generateInterceptors(): (options: any, nextCall: any) => grpc.InterceptingCall {
```

Comment:
Please add a comment on what is the use of this.

Reply:
```suggestion
export function getApiTokenClientInterceptor(): (options: any, nextCall: any) => grpc.InterceptingCall {
```

Reply:
You can move this to index.ts too.

Reply:
rename the method and moved to index.ts